### ISSUE TITLE
Pin OpenMM version >=7.3.1,<7.6

### ISSUE THREAD

**Issue by mjw99** [conda-forge/openmmtools-feedstock#9]
Due to a recent top level Python package rename in OpenMM 7.6, some modules will no longer be found. I think this is the correct syntax:
```
    - openmm >=7.3.1,<7.6
```
See also:
https://github.com/choderalab/yank/issues/1260
https://github.com/choderalab/yank/issues/1261

**Comment by jchodera** [comment]
@ijpulidos @mikemhenry : If these are really fatal errors rather than just warnings, we should focus on getting updated bugfix releases out of these packages (yank, openmmtools) that fix the imports. Can you help here?

In the meantime, it does sound reasonable to pin the useful versions of these to openmm<7.6 (as @mjw99 suggests) if these are indeed errors and not warnings.

**Comment by mikemhenry** [comment]
I will investigate! I thought we fixed things in a backwards compatible way with the new namespace changes

**Comment by mikemhenry** [comment]
Plan is to push out a bug fix release, then once openmm 7.7 is out, we will bump the build number and prevent openmm 7.6 being pulled in which we know has issues: https://github.com/openmm/openmm/issues/3310

**Comment by mjw99** [comment]
Many thanks for your help Mike!

**Comment by mikemhenry** [comment]
@mjw99 Just wanted to give you an update, we are working on a blocking issue here https://github.com/choderalab/openmmtools/pull/538 and once we get that merged, we will be cutting a new release that will fix this. Thank you for your patience.

**Comment by ijpulidos** [comment]
Yes, as Mike says we are currently working on solving the critical issues with how `ContextCache` is being used and getting very close to gettting something completed.

My opinion on this one is that we should pin to versions equal or larger than the minimum version we aim to support from OpenMM or not pinning at all (just `openmm`). I think this is what has less issues for future releases of the dependencies.

**Comment by mjw99** [comment]
Dear @mikemhenry  and @ijpulidos,
I really appreciate all your hard work here and the updates.

Many thanks,

Mark
